### DISCUSSION TITLE
How to get a pull request?

### DISCUSSION THREAD

**Discussion by siliconvolley** [community/community#63173] [Discussions]
### Select Topic Area

Question

### Body

Pull Request

**Comment by Ananyapk25** [comment]
At first fork the repo, then create a branch
then make changes in that branch
then commit
and then go for contribute and do a pull request
to main branch

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
